README.md

```diff
@@ -145,11 +145,11 @@ pusher.trigger("test_channel", "my_event", &hash_map).await;
 
 #### Multiple channels
 
-##### `async fn trigger_multi<S: serde::Serialize>(&self, channels: &Vec<&str>, event: &str, payload: S)`
+##### `async fn trigger_multi<S: serde::Serialize>(&self, channels: &[&str], event: &str, payload: S)`
 
 |Argument | Description |
 |:-:|:-:|
-|channels `&Vec<&str>`| A vector of channel names you wish to send an event on. The maximum length is 10.|
+|channels `&[&str]`| A vector of channel names you wish to send an event on. The maximum length is 10.|
```

Comment:
Is this a breaking change?

Reply:
No - Since the `Vec` is not mutable this is equivalent and is more idiomatic.